packages/functional-tests/tests/oauth/resetPassword.spec.ts

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, expect } from '../../lib/fixtures/standard';
+import { EmailHeader, EmailType } from '../../lib/email';
+
+let email;
+const password = 'passwordzxcv';
+const newPassword = 'Newpassword@';
+
+test.describe('oauth reset password', () => {
+  test.beforeEach(async ({ target, pages: { login } }) => {
```

Comment:
Why do we need to create an account here? Are we able to use the `credentials.email` from the test?

Reply:
modified to use `credentials.email`